packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -341,7 +343,14 @@ - (int64_t)position {
   return FLTCMTimeToMillis([_player currentTime]);
 }
 
+- (bool)isDurationIndefinite {
+  return CMTIME_IS_INDEFINITE([[_player currentItem] duration]);
```

Comment:
Based on the comment in the PR, CMTIME_IS_INDEFINITE returns true when duration is 0. 
Then this below code would never be true if we land this PR
https://github.com/flutter/plugins/blob/master/packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m#L316

This is expected?

Reply:
CMTIME_IS_INDEFINITE becomes simplified to '0' within `FLTCMTimeToMillis` which is being used within `[self duration]`. In this case I'm calling CMTIME_IS_INDEFINITE on `[[_player currentItem] duration]` which is still a CMTime representation of the duration.

There can be instances where a CMTime.value = 0 but is not CMTIME_IS_INDEFINITE so `[self duration]` can return 0 and the code at [#L316](https://github.com/flutter/plugins/blob/master/packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m#L316) can still be true in certain cases.